### PR TITLE
SISRP-15412 - Enrollment card is not showing on calcentral-sis-dev

### PR DESCRIPTION
**Pull request by christianvuerings**
https://jira.berkeley.edu/browse/SISRP-15412

### CODE REVIEW

**Comment by paulkerschen** [comment]
@christianvuerings : We'll also need to change `affiliations.json`, `person_edo.json` and `student_edo.json` in `fixtures/json`. If our tests are working as designed, we should get a number of Travis failures without those changes. :)

**Comment by christianvuerings** [comment]
@paulkerschen updated fixtures & tests, fingers crossed.

File: fixtures/json/affiliations.json [hunk 1]
```diff
@@ -19,7 +19,10 @@
                 "code": "APPLICANT",
                 "description": "Applicant"
               },
-              "statusCode": "INA",
+              "status": {
+                "code": "INA",
+                "description": "Inactive"
+              },
               "statusDescription": "Inactive",
```
Comment (paulkerschen) [inline fixtures/json/affiliations.json]: `statusDescription` key should come out here, and in three places below.

**Comment by paulkerschen** [comment]
Thanks @christianvuerings; merging with a note that the `statusDescription` elements should be removed from fixtures in a follow-up.

**Comment by christianvuerings** [comment]
@paulkerschen thanks - follow up at #4938

**Pull request merged**